spec/controllers/users_controller_spec.rb

```diff
@@ -103,10 +103,24 @@
 
     context "when user attempts to edit another profile" do
       let(:another_user) { FactoryGirl.create(:user) }
-      it "redirects to show profile" do
-        get :edit, id: another_user.user_key
-        expect(response).to redirect_to(@routes.url_helpers.profile_path(another_user.to_param))
-        expect(flash[:alert]).to include("Permission denied: cannot access this page.")
+      context 'with default abilities' do
+        it "redirects to show profile" do
+          expect_any_instance_of(Ability).to receive(:can?).with(:edit, another_user).and_return(false)
```

Comment:
Yes this test still passes without this line. expect_any_instance_of is an expect statement.  It is not mocking Ability. It is asserting the default abilities.